src/BinOp.chpl

```diff
@@ -1185,6 +1217,22 @@ module BinOp
         }
       var repMsg = "created %s".doFormat(st.attrib(rname));
       return new MsgTuple(repMsg, MsgType.NORMAL);
+    } else if (r.etype == bool && val.type == bool)  {
+      select op {
+          when "<<" {
+            e.a = val:int << r.a:int;
+          }
+          when ">>" {
+            e.a = val:int >> r.a:int;
+          }
```

Comment:
It's not necessary but we could wrap these in an `if val` block to skip doing anything when `val` is false. Chapel zeroes out arrays when they are created and false shifted in either direction would remain 0, so we could just leave `e.a` as it is

But that's optimizing for a case that's prob not terribly common and isn't terribly expensive so maybe it would be better to just avoid the clutter

Reply:
Made the change.  Thanks.